app/javascript/dashboard/routes/dashboard/helpcenter/components/PortalListItem.vue

```diff
@@ -0,0 +1,415 @@
+                'HELP_CENTER.PORTAL.PORTAL_SETTINGS.LIST_ITEM.PORTAL_CONFIG.TITLE'
+              )
+            }}
+          </h2>
+          <div class="configuration-items--wrap">
+            <div class="configuration-items">
+              <div class="configuration-item">
+                <span class="heading">{{
+                  $t(
+                    'HELP_CENTER.PORTAL.PORTAL_SETTINGS.LIST_ITEM.PORTAL_CONFIG.ITEMS.NAME'
+                  )
+                }}</span>
+                <span class="content">{{ portal.header_text }}</span>
+              </div>
+              <div class="configuration-item">
+                <span class="heading">{{
+                  $t(
+                    'HELP_CENTER.PORTAL.PORTAL_SETTINGS.LIST_ITEM.PORTAL_CONFIG.ITEMS.DOMAIN'
+                  )
+                }}</span>
+                <span class="content">{{ portal.custom_domain }}</span>
+              </div>
+            </div>
+            <div class="configuration-items">
+              <div class="configuration-item">
+                <span class="heading">{{
+                  $t(
+                    'HELP_CENTER.PORTAL.PORTAL_SETTINGS.LIST_ITEM.PORTAL_CONFIG.ITEMS.SLUG'
+                  )
+                }}</span>
+                <span class="content">{{ portal.slug }}</span>
+              </div>
+              <div class="configuration-item">
+                <span class="heading">{{
+                  $t(
+                    'HELP_CENTER.PORTAL.PORTAL_SETTINGS.LIST_ITEM.PORTAL_CONFIG.ITEMS.TITLE'
+                  )
+                }}</span>
+                <span class="content">{{ portal.page_title }}</span>
+              </div>
+            </div>
+            <div class="configuration-items">
+              <div class="configuration-item">
+                <span class="heading">{{
+                  $t(
+                    'HELP_CENTER.PORTAL.PORTAL_SETTINGS.LIST_ITEM.PORTAL_CONFIG.ITEMS.THEME'
+                  )
+                }}</span>
+                <div class="content-theme-wrap">
+                  <div
+                    class="theme-color"
+                    :style="{ background: portal.color }"
+                  />
+                  <span class="content">{{ portal.page_title }}</span>
+                </div>
+              </div>
+              <div class="configuration-item">
+                <span class="heading">{{
+                  $t(
+                    'HELP_CENTER.PORTAL.PORTAL_SETTINGS.LIST_ITEM.PORTAL_CONFIG.ITEMS.SUB_TEXT'
+                  )
+                }}</span>
+                <span class="content">{{ portal.page_title }}</span>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="portal-locales">
+          <h2 class="locale-title">
+            {{
+              $t(
+                'HELP_CENTER.PORTAL.PORTAL_SETTINGS.LIST_ITEM.AVAILABLE_LOCALES.TITLE'
+              )
+            }}
+          </h2>
+          <table>
```

Comment:
Can we move this table as a separate component.

Reply:
Will do it

Reply:
Done https://github.com/chatwoot/chatwoot/pull/5146/commits/dfb0fee4025c93e61807f42e2afe3a6b802fc038